mmv1/products/compute/api.yaml

```diff
@@ -9741,6 +9741,66 @@ objects:
                     - :NEVER
                     - :ON_PERMANENT_INSTANCE_DELETION
                   default_value: :NEVER
+          - !ruby/object:Api::Type::Map
+            name: 'internalIPs'
```

Comment:
would changing that to internalIps make the fields users create be internal_ips?